component/src/main/java/me/androidbox/component/general/AgendaBottomSheet.kt

```diff
@@ -0,0 +1,77 @@
+package me.androidbox.component.general
+
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material3.ExperimentalMaterial3Api
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.ModalBottomSheet
+import androidx.compose.material3.rememberModalBottomSheetState
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.rememberCoroutineScope
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import kotlinx.coroutines.launch
+import me.androidbox.component.R
+import me.androidbox.component.ui.theme.BusbyTaskyTheme
+import me.androidbox.component.ui.theme.backgroundWhiteColor
+
+@OptIn(ExperimentalMaterial3Api::class)
+@Composable
+fun AgendaBottomSheet(
+    shouldOpenBottomSheet: Boolean,
+    onCloseDropdown: () -> Unit,
+    content: @Composable () -> Unit,
+    modifier: Modifier = Modifier) {
+
+    val bottomSheetState = rememberModalBottomSheetState(skipPartiallyExpanded = true)
+    val bottomSheetScope = rememberCoroutineScope()
+
+    if(shouldOpenBottomSheet) {
```

Comment:
This if condition shouldn't be there, since the bottom sheet state already manages if it's shown or not. That's also the reason it disappears without animation when setting this state to false

Reply:
@philipplackner By removing the condition the bottom sheet will show when entering the screen. Also the bottomsheet doesn't completely disappear and the top part of it is still showing and cannot interact with the screen as the bottomsheet is still active. In red box in the image below.

![bottomhiding](https://user-images.githubusercontent.com/14260802/236692577-e4430434-61b3-4ccf-b480-e9a319bb203f.png)